client/my-sites/upgrades/domain-management/dns/dns-record.jsx

```diff
@@ -91,6 +91,10 @@ var DnsRecord = React.createClass( {
 	},
 
 	render: function() {
+		const isBeingDeleted = this.props.dnsRecord.isBeingDeleted && this.props.dnsRecord.isBeingDeleted === true;
```

Comment:
Heh, you're right, I was trying to be extra careful, but I got too carried away :grin: